backend/flindt/integrations/importer.py

```diff
@@ -51,7 +51,8 @@ def import_users(self):
                 ...
             ]}
         """
-        all_users = self._make_api_request('people').get('people')
+        users_from_circle = 'people?circle_id={}'.format(self.organization.glassfrog_anchor_circle_id)
```

Comment:
Gaat dit kapoet als we geen anchor circle invullen?

Reply:
Jep, zal even check inbouwen